### PR TITLE
VPA histogram percentile should return end of the bucket instead of the middle point

### PR DESCRIPTION
**Pull request by schylek**

### CODE REVIEW

File: vertical-pod-autoscaler/pkg/recommender/util/histogram.go [hunk 1]
```diff
@@ -164,9 +164,8 @@ func (h *histogram) Percentile(percentile float64) float64 {
 	}
 	bucketStart := h.options.GetBucketStart(bucket)
```
Comment (mwielgus) [inline vertical-pod-autoscaler/pkg/recommender/util/histogram.go]: this variable is not needed.

**Comment by mwielgus** [comment]
1 comment, otherwise lgtm

File: vertical-pod-autoscaler/pkg/recommender/util/histogram.go (hunk 1 above)
Comment (schylek) [reply vertical-pod-autoscaler/pkg/recommender/util/histogram.go]: Done.

**Comment by kgolab** [comment]
LGTM

**Pull request merged**